js/apps/system/_admin/aardvark/APP/frontend/js/views/documentView.js

```diff
@@ -536,8 +536,9 @@
 
     breadcrumb: function () {
       var name = window.location.hash.split('/');
+      var collid = decodeURIComponent(name[0]);
       $('#subNavigationBar .breadcrumb').html(
-        '<a href="#collection/' + name[1] + '/documents/1">Collection: ' + _.escape(name[1].length > 64 ? name[1].substr(0, 64) + "..." : name[1]) + '</a>' +
+        '<a href="#collection/' + name[1] + '/documents/1">Collection: ' + _.escape(collid.length > 64 ? collid.substr(0, 64) + "..." : collid) + '</a>' +
```

Comment:
Previously, `collid` was referring to `name[1]`, now it's `name[0]`. Is that intentional?

Reply:
Good catch. Fixed.